src/reactHotLoader.js

```diff
@@ -16,7 +16,15 @@ import logger from './logger'
 
 import { preactAdapter } from './adapters/preact'
 
-function resolveType(type) {
+const forceSimpleSFC = { proxy: { allowSFC: false } }
```

Comment:
React.memo could "eat" only simple class or functional component.
The current approach, with "indeterminate" components is breaking the stuff.